app/src/main/AndroidManifest.xml

```diff
@@ -62,6 +65,29 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
             </intent-filter>
+
+            <intent-filter android:label="@string/app_name">
```

Comment:
Instead of  app name we can use a specific name here for cases such as "Open referendum", "Import account" etc.